awx/ui_next/src/screens/Job/JobDetail/JobDetail.jsx

```diff
@@ -58,6 +60,16 @@ const VERBOSITY = {
   4: '4 (Connection Debug)',
 };
 
+const getJobModel = type => {
+  if (type === 'ad_hoc_command') return AdHocCommandsAPI;
+  if (type === 'inventory_update') return InventoriesAPI;
+  if (type === 'project_update') return ProjectUpdatesAPI;
```

Comment:
hey @jakemcdermott, I got this code locally, and when attempt to `cancel` a project update on the job details page, and I am getting the following error. The same error does not appear if I cancel the job on the job output.  
<img width="1785" alt="image" src="https://user-images.githubusercontent.com/9053044/112147889-98915400-8bb3-11eb-8980-03b3b53e1883.png">

Reply:
@nixocio 

:facepalm: Looks like the `cancel` methods aren't actually defined on all of the models yet. I'll fix this shortly.

Reply:
https://github.com/ansible/awx/pull/9659/commits/72d700e12587b3dacba79c91b983ce5a6fac5c5f